Doc/whatsnew/3.14.rst

```diff
@@ -531,10 +531,13 @@ unittest
 uuid
 ----
 
-* Add support for UUID version 8 via :func:`uuid.uuid8` as specified
+* Add support for UUID version 6 via :func:`uuid.uuid6` as specified
   in :rfc:`9562`.
   (Contributed by Bénédikt Tran in :gh:`89083`.)
 
+* Add support for UUID version 8 via :func:`uuid.uuid8` as specified
+  in :rfc:`9562`.
+  (Contributed by Bénédikt Tran in :gh:`89083`.)
```

Comment:
We could combine these? Eventually we can say "versions 6-8":
```suggestion
* Add support for UUID versions 6 and 8 via :func:`~uuid.uuid6` and 
  :func:`~uuid.uuid8`,  as specified in :rfc:`9562`.
  (Contributed by Bénédikt Tran in :gh:`89083`.)
```

Reply:
Let's do it like that indeed. I hope we'll be able to add UUIDv7 before it's feature frozen (the discussion is still pending on the issue).